tests/shared/src/test/scala/cats/tests/KleisliSuite.scala

```diff
@@ -127,12 +127,29 @@ class KleisliSuite extends CatsSuite {
   checkAll("Kleisli[Option, MiniInt, *]", AlternativeTests[Kleisli[Option, MiniInt, *]].alternative[Int, Int, Int])
   checkAll("Alternative[Kleisli[Option, Int, *]]", SerializableTests.serializable(Alternative[Kleisli[Option, Int, *]]))
 
-  checkAll("Kleisli[Const[String, *], MiniInt, *]",
-           ContravariantMonoidalTests[Kleisli[Const[String, *], MiniInt, *]].contravariantMonoidal[Int, Int, Int]
-  )
-  checkAll("ContravariantMonoidal[Kleisli[Option, Int, *]]",
-           SerializableTests.serializable(ContravariantMonoidal[Kleisli[Const[String, *], Int, *]])
-  )
+  {
+    checkAll("Kleisli[Predicate, MiniInt, MiniInt]",
+             DecidableTests[Kleisli[Predicate, MiniInt, *]].decidable[MiniInt, MiniInt, MiniInt]
+    )
+    checkAll(
+      "Decidable[Kleisli[Option, Int, *]]",
+      SerializableTests.serializable[Decidable[Kleisli[Predicate, MiniInt, *]]](
+        Decidable[Kleisli[Predicate, MiniInt, *]]
+      )
+    )
+  }
+
+  {
+    implicit val catsDataContravariantMonoidalForKleisliLocal
+      : ContravariantMonoidal[Kleisli[Const[String, *], Int, *]] =
```

Comment:
Why does this `implicit` need to be scoped like this, is there some sort of ambiguity issue?

Reply:
Doesn't look like it needs to be - probably just cargo culting when I wrote it.